pkg/proc/bininfo.go

```diff
@@ -1111,8 +1109,20 @@ func (bi *BinaryInfo) openSeparateDebugInfo(image *Image, exe *elf.File, debugIn
 			break
 		}
 	}
+	// We cannot find the debug information locally on the system. Try and see if we're on a system that
+	// has debuginfod so that we can use that in order to find any relevant debug information.
 	if debugFilePath == "" {
-		return nil, nil, ErrNoDebugInfoFound
+		const debuginfodFind = "debuginfod-find"
+		if _, err := exec.LookPath(debuginfodFind); err == nil {
+			cmd := exec.Command(debuginfodFind, "debuginfo", desc1+desc2)
+			out, err := cmd.CombinedOutput()
+			if err != nil {
+				return nil, nil, err
```

Comment:
```
return nil, nil, fmt.Errorf("could not open debug info, additionally debuginfod-find returned %v %q", err, string(out))
```

otherwise the error is going to be "exit status 1", or something like that.

Reply:
or maybe we should just return ErrNoDebugInfoFound and log the output of debuginfod-find instead. It's more likely that we end up in this path because the executable is stripped accidentally than because debuginfod-find could have helped but didn't.

Reply:
Done.